state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -386,7 +386,7 @@ impl DataSummaryPoller {
     }
 
     pub async fn start(self) {
-        trace!("DataSummaryPoller: start");
+        trace!("Starting the diem data poller!");
```

Comment:
this message seems more like info?

Reply:
Yeah, I believe Philip added these logs only for debugging. My plan is to get @khiemngo to add structured logging to the entire of the Diem Data Client, so this will be fixed in the future. @khiemngo, let's sync on getting this started once this PR lands 😄